website/source/guides/governance-and-policy/sentinel/sentinel-policy.html.markdown

```diff
@@ -194,6 +194,10 @@ easy to read and write, while being fast to evaluate. There is no limitation on
 how complex policies can be, but they are in the execution path so care should
 be taken to avoid adversely impacting performance.
 
+The Nomad integration for Sentinel implements all of the available [standard
```

Comment:
the "why not" would also be useful here

Reply:
Hi! The `http` import is intended for Sentinel integrations like Terraform where policy checks don't run directly in the request path as they do in Nomad. I'll add a small note here for posterity.

Reply:
@jippi We'd consider enabling it in the future, but for a product like Nomad it's going to take a lot of careful consideration so people don't accidentally expose internal endpoints they shouldn't during sentinel policy evaluation.